dev/storage.h

```diff
@@ -142,33 +142,72 @@ namespace sqlite_orm {
                 perform_void_exec(db, ss.str());
             }
 #endif
+            void add_generated_cols(std::vector<table_xinfo*>& columnsToAdd,
+                                    std::vector<table_xinfo>& storageTableInfo) {
```

Comment:
if `storageTableInfo` is not modified within `add_generated_cols` please pass it as const ref instead

Reply:
done